src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -256,10 +256,19 @@ public interface IDurableOrchestrationClient
         /// </summary>
         /// <param name="instanceId">The ID of the orchestration instance to query.</param>
         /// <param name="showHistory">Boolean marker for including execution history in the response.</param>
-        /// <param name="showHistoryOutput">Boolean marker for including input and output in the execution history response.</param>
+        /// <param name="showHistoryOutput">Boolean marker for including output in the execution history response.</param>
         /// <param name="showInput">If set, fetch and return the input for the orchestration instance.</param>
```

Comment:
If we follow my suggestion and reinterpret `showInput` to apply to both the orchestration status and the history items, then we can revert all the changes to this file. However, we will need to update the `showInput` description here to say: `If set, fetch and return the input for the orchestration instance. If both <c>showHistory</c> and <see cref="DurableTaskOptions.StoreInputsInOrchestrationHistory"/> are also set to <c>true</c>, then the inputs for activity and sub-orchestration events in the orchestration history will also be returned.`